apps/src/code-studio/peer_reviews/PeerReviewSubmissionData.story.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PeerReviewSubmissionData from './PeerReviewSubmissionData';
+import reactBootstrapStoryDecorator from '../pd/reactBootstrapStoryDecorator';
+
+export default storybook => {
+  storybook
+    .storiesOf('Peer Review Submissions', module)
+    .addDecorator(reactBootstrapStoryDecorator)
+    .addStoryTable([
+      {
+        name: 'Peer Review Submissions',
+        story: () => (
+          <PeerReviewSubmissionData
+            filterType="escalated"
+            submissions={[
+              {
+                submitter: 'Jon Snow',
+                course_name: 'Intro to knowing things',
+                unit_name: 'So you know nothing',
+                level_name: 'Documenting all that you know',
+                submission_date: '2/19/2017',
+                escalation_date: '4/3/2/2017',
+                id: 2
```

Comment:
does it matter that the ids for both of these are 2? (I'm not sure how storybook works so I don't know what these ids mean)

Reply:
Will fix